gradle/versions.gradle

```diff
@@ -277,6 +279,7 @@ ext {
         stax2_api                          : "org.codehaus.woodstox:stax2-api:4.2.1",
         hamcrest                           : "org.hamcrest:hamcrest:2.2",
         aopalliance_repackaged             : "org.glassfish.hk2.external:aopalliance-repackaged:3.0.3",
+        osgi_resource_locator              : "org.glassfish.hk2:osgi-resource-locator:2.4.0",
```

Comment:
where is this being used?

Reply:
its used in onboarding-enabler-java-sample-app: